### PR TITLE
[ODS-4477] Docker File for deploying ODS / API running with the Kestrel web server

### PR DESCRIPTION
**Pull request by ptar-innive**

### CODE REVIEW

File: WebApi/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,8 @@
 FROM mcr.microsoft.com/dotnet/core/runtime:3.1-alpine
 LABEL maintainer="Ed-Fi Alliance, LLC and Contributors <techsupport@ed-fi.org>"
-RUN apk update && apk upgrade
\ No newline at end of file
+ENV VERSION="5.1.0-b12868"
+RUN apk update && apk upgrade && apk add unzip
+WORKDIR /app
+RUN wget -O /app/WebApi.zip https://www.myget.org/F/ed-fi/api/v2/package/EdFi.Suite3.Ods.WebApi/${VERSION} && unzip /app/WebApi.zip -d /app && rm -f /app/WebApi.zip
+EXPOSE 80
+ENTRYPOINT ["dotnet","EdFi.Ods.WebApi.exe"]
```
[the remark this answers is not in the document]
Comment (jamessmckay) [reply WebApi/Dockerfile:8]: we also need to pass in an environment as a command line arguemnt for the application for example we do this by adding:
`--environment Production` for the production environment (Development for the development environment).

**Review by jamessmckay — CHANGES_REQUESTED** [review]
I would like to sidebar this ticket in the morning.

File: WebApi/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,8 @@
 FROM mcr.microsoft.com/dotnet/core/runtime:3.1-alpine
 LABEL maintainer="Ed-Fi Alliance, LLC and Contributors <techsupport@ed-fi.org>"
-RUN apk update && apk upgrade
\ No newline at end of file
+ENV VERSION="5.1.0-b12868"
```
[the remark this answers is not in the document]
Comment (ptar-innive) [reply WebApi/Dockerfile:3]: i would not suggest it, during running the container there is no sense of it, as the code is there in the container, during build time this env variable is a clear indication of which version of software is running in the container, if you can modify it without commit, then how would you know what version was build? Images should be atomic

File: WebApi/Dockerfile [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+RUN apk update && apk upgrade && apk add unzip
+WORKDIR /app
+RUN wget -O /app/WebApi.zip https://www.myget.org/F/ed-fi/api/v2/package/EdFi.Suite3.Ods.WebApi/${VERSION} && unzip /app/WebApi.zip -d /app && rm -f /app/WebApi.zip
```
[the remark this answers is not in the document]
Comment (ptar-innive) [reply WebApi/Dockerfile:6]: is not that come from application.json which is a volume file?

File: WebApi/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (ptar-innive) [reply WebApi/Dockerfile]: how many such startup options should we handle? do it have documentation about it? Does it came from application.json?

**Review by jamessmckay — APPROVED** [review]

**Pull request merged**